### PR TITLE
K8SPSMDB-341 - Increment versions after 1.6.0 release

### PR DESCRIPTION
**Pull request by tplavcic**
[K8SPSMDB-341](https://jira.percona.com/browse/K8SPSMDB-341)

Two backup tests are failing because of: https://github.com/percona/percona-server-mongodb-operator/pull/515
`upgrade-consistency` is fixed in the second commit

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| init-deploy | passed |
| storage | passed |
| upgrade-consistency | failed |
| monitoring-2-0 | passed |
| self-healing | passed |
| limits | passed |
| demand-backup | failed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| rs-shard-migration | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| scheduled-backup | failed |

File: e2e-tests/upgrade-consistency/run [hunk 1]
```diff
@@ -80,8 +80,14 @@ main() {
     }'
     wait_for_running "${CLUSTER}-rs0" "1" "false"
 
-    compare_kubectl service/${CLUSTER}-rs0 "-160"
-    compare_kubectl statefulset/${CLUSTER}-rs0 "-160"
```
Comment (delgod) [inline e2e-tests/upgrade-consistency/run:84]: please don't remove these lines

**Review by delgod — CHANGES_REQUESTED** [review]

File: e2e-tests/upgrade-consistency/run (hunk 1 above)
Comment (tplavcic) [reply e2e-tests/upgrade-consistency/run]: nice catch! fixed.

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | passed |
| self-healing | passed |
| demand-backup | failed |
| limits | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| service-per-pod | passed |
| rs-shard-migration | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| scheduled-backup | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| init-deploy | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| demand-backup | failed |
| monitoring-2-0 | passed |
| limits | passed |
| scaling | passed |
| arbiter | passed |
| operator-self-healing | passed |
| security-context | passed |
| rs-shard-migration | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| scheduled-backup | failed |

**Pull request merged**